corehq/apps/userreports/management/commands/prune_old_datasources.py

```diff
@@ -40,17 +40,20 @@ def handle(self, **options):
         for engine_id, expected_tables in tables_by_engine_id.items():
             engine = connection_manager.get_engine(engine_id)
             with engine.begin() as connection:
-                results = connection.execute("""
+                # Using string formatting rather than execute with %s syntax
+                # is acceptable here because the strings we're inserting are static
+                # and only templated for DRYness
+                results = connection.execute(f"""
                 SELECT table_name
                   FROM information_schema.tables
                 WHERE table_schema='public'
                   AND table_type='BASE TABLE'
                   AND (
-                    table_name LIKE '{}%%'
+                    table_name LIKE '{UCR_TABLE_PREFIX}%%'
```

Comment:
You don't need the double percent signs anymore, right?

Reply:
Mmm, I think you do. It's confusing but I think I can make a convincing case. I think the first hint is that I didn't go from `%` syntax to `{}` syntax in the change, I just went from one `{}`-based syntax to another, so I don't think it'd affect the behavior of `%`. But that doesn't answer why you need to escape `%` as `%%`. I think the reason for that is that the first argument of execute is always used as an `%`-based string template. Here we didn't pass any explict params so the call ends up being basically `template % []`, but normally you'd pass values you want `execute` to escape as a list of params in the second argument.

Reply:
> the first argument of execute is always used as an %-based string template

Interesting, I didn't know that. Thanks for explaining.